common/xdc/nDCHistoryRereplicator.go

```diff
@@ -0,0 +1,217 @@
+// Copyright (c) 2019 Uber Technologies, Inc.
```

Comment:
can we plz change this to nDCHIstoryResender?

Reply:
done